pubsub/google/cloud/pubsub_v1/publisher/batch/thread.py

```diff
@@ -204,6 +204,8 @@ def _commit(self):
                 self._messages,
             )
             end = time.time()
+            self._client._publish_count += len(self._messages)
+            self._client._commit_count += 1
```

Comment:
Seems like this bit should be encapsulated by a method on the client.

Reply:
got it.